scenario-api/src/main/scala/pl/touk/nussknacker/engine/graph/evaluatedparam.scala

```diff
@@ -3,6 +3,7 @@ package pl.touk.nussknacker.engine.graph
 import io.circe.generic.JsonCodec
 import pl.touk.nussknacker.engine.graph.expression.Expression
 
+// TODO: rename to NodeParameter
```

Comment:
the import renames are inconsistent, some are `=> NodeParameter`, some not renamed, and one is even `=> EvaluatedParameter` (intellij `usage in import` usage category is handy for this)

i'm aware that import renames are a termporary solution, but i think it's better to make them consistent

of course this will be hard to maintain (always remember to rename import), but it's probably better to degenerate into an inconsistent state than start in it

Reply:
same goes for nussknacker.engine.api.expression

Reply:
I intended to change only places where both definition (Parameter) and NodeParameter were used altogether, but you're right - I changed all places for the sake of consistency.

Reply:
nice, but did you intentionally not do it for nussknacker.engine.api.expression.Expression aka CompiledExpression? it's import renames are also inconsistent

Reply:
You're right again - fixed